src/components/button.js

```diff
@@ -1,22 +1,22 @@
-import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+// import { useDispatch } from 'react-redux';
+// import { removeBookFromApi } from '../redux/books/books';
 
-const Button = (props) => {
-  const dispatch = useDispatch();
+// const Button = (props) => {
+//   const dispatch = useDispatch();
 
-  const bookToRemove = (e) => {
-    const smbook = {
-      id: e.target.id,
-    };
+//   const bookToRemove = (e) => {
+//     const smbook = {
+//       id: e.target.id,
+//     };
 
-    dispatch(removeBook(smbook));
-  };
-  const id = props;
-  return (
-    <button type="button" onClick={bookToRemove} id={id.id}>
-      Remove Book
-    </button>
-  );
-};
+//     dispatch(removeBookFromApi(smbook));
+//   };
+//   const id = props;
+//   return (
+//     <button type="button" onClick={bookToRemove} id={id.id}>
+//       Remove Book
+//     </button>
+//   );
+// };
```

Comment:
Let's remove all commented code, so as to make the code cleaner ✊ 

Kindly check for other occurrences 🙂